### ISSUE TITLE
Not playing anything

### ISSUE THREAD

**Issue by dCo3lh0** [son-link/PQMusic#9]
I did install its dependencies, the application opens, but when i open music with it, the slider dont start and dont move from beginning, the play icon becomes pause, i think its not starting anything, this here is the error if i start by terminal.

`defaultServiceProvider::requestService(): no service found for - "org.qt-project.qt.mediaplayer"`

**Comment by dCo3lh0** [comment]
Just giving a heads up here, the last version that did work for me was 1.2.0, after this i dont know what you did, but stopped working for me, I'm on a LMDE5 based on Debian 11, using Kernel 6.1.20 and Cinnamon 5.6.8

**Comment by son-link** [comment]
Are you using the AppImage, or have you installed it via pip, source code or AUR?

**Comment by dCo3lh0** [comment]
Im using appimage, should i try the pip version? gonna try here

**Comment by dCo3lh0** [comment]
Yeah, using the Pip installed version it works fine as well, but it doesnt follow themes or have themes. I'm on cinnamon 5.6.8 which is gnome gtk, some qt5 apps get themes, some dont. But its working.

**Comment by son-link** [comment]
For the themes you need to install a package called qt5ct, which allows you to configure the look and feel of Qt5 programs when not using Plasma, LXQt or any other Qt-integrated environment. I don't know if the package is available in the LMDE repositories, but I'll leave you the link in case you want to compile it yourself: https://sourceforge.net/projects/qt5ct/

As for the error, I see it's due to missing Qt5 libraries for multimedia, which is strange because the PyPi package comes with them. Then I'll download a distribution that doesn't use a Qt-based environment for testing.

**Comment by dCo3lh0** [comment]
yeah im using Qt5ct, i have a fusion darker style applied here, some applications follow qt5ct theme and some dont. Your AppImage follows the embedded theme in it, but pip version dont.
